config/routes.rb

```diff
@@ -12,8 +12,14 @@
 
   mount SwaggerUiEngine::Engine, at: '/api'
 
+  def swagger_base_path
+    @swagger_base_path ||= YAML.load_file(Rails.root.join("public/doc/swagger-2.yaml"))["basePath"]
+  end
+
   def add_swagger_route http_method, path, opts = {}
-    full_path = path.gsub(/{(.*?)}/, ':\1')
+    full_path = path.gsub(/{(.*?)}/, ':\1')             # Covert codegen substitutions into Symbols
+    full_path = full_path.split(swagger_base_path).last # Remove the basePath that codegen inserts
```

Comment:
For QE, I was told they are controlling the base path, so that is expected.
For Dev hitting the API directly, they will need this client change.
For Dev hitting the proxy, IMO the proxy should handle it, but for now, I remember we discussed adding an env var, so the javascript client can still connect using the base path.  I'll update the PR for that.